LABTASKS.md

```diff
@@ -0,0 +1,90 @@
+# Lab Tasks <!-- omit in toc -->
+
+- [Getting started](#getting-started)
+- [Exploring the project](#exploring-the-project)
+- [Overview of the lab](#overview-of-the-lab)
+- [Workflow and Continuous Integration](#workflow-and-continuous-integration)
+- [Writing Todos to the Database](#writing-todos-to-the-database)
+- [Display Todos](#display-todos)
+
+## Getting started
+
+- Remember to set up ZenHub with your stories and estimates
+- Turn in the URL of the GitHub repository for your group in Canvas. This will make it easier for us to figure out which team is "Snoozing Llamas".
+
+Definitely ask if you're ever confused about what you need to do for a given task.
+
+## Exploring the project
+
+The structure of this project should be nearly identical to that of lab #3, and as such there really isn't much excitement in that department.
+
+The server is, for the most part, the same as it has been in the past two labs. The difference to look for here is in how the server gets the data it sends out in reply to requests.
+
+Spend some time looking over the project with these questions in mind. (Unlike the previous labs, however, you do _not_ need to write up and turn in your answers to these questions.)
+
+1. What do we do in the `Server` and `UserController` constructors
+   to set up our connection to the development database?
+1. How do we retrieve a user by ID in the `UserController.getUser(String)` method?
+1. How do we retrieve all the users with a given age
+   in `UserController.getUsers(Context...)`? What's the role of
+   the variable `filters` in that method?
+1. What is happening in the `UserControllerSpec.setUpEach()` method?
+1. What's being tested in `UserControllerSpec.GetUsersByAge()`?
+   How is that test implemented?
+1. Follow the process for adding a new user. What role does `UserController` play in the process?
+
+## Overview of the lab
+
+- Re-implement the ToDo API, this time pulling data from MongoDB rather than from a flat JSON file. The new elements of this lab are primarily in the MongoDB interaction, which is mostly in the Java server code.
```

Comment:
```suggestion
- Re-implement the Todo API, this time pulling data from MongoDB rather than from a flat JSON file. The new elements of this lab are primarily in the MongoDB interaction, which is mostly in the Java server code.
```